udp-reassembly-offload.md

```diff
@@ -27,6 +27,25 @@ This section describes necessary updates in the Windows network stack to support
 
 The NDIS interface for URO is used for communication between TCPIP and the NDIS miniport driver.
 
+## Rules
+
+URO can only be attempted on a batch of packets that meet all the following criteria:
+
+- Unicast only (no broadcast or multicast)
+- 5-tuple matches
+- Payload length is identical for all datagrams
```

Comment:
This is not a requirement for URO in general. Do we see a reason why a final datagram cannot be smaller, if the HW/driver supports it? Agree we should not mandate coalescing different-sized datagrams.